S2VX.Game/S2VXScrollContainer.cs

```diff
@@ -0,0 +1,10 @@
+﻿using osu.Framework.Graphics.Containers;
+using osu.Framework.Input.Events;
+
+namespace S2VX.Game {
+    public class S2VXScrollContainer : BasicScrollContainer {
+        public S2VXScrollContainer() { }
+
+        protected override bool OnScroll(ScrollEvent e) => false;
```

Comment:
How is this going to work? You're basically removing all the scrolling functionality of the `BasicScrollContainer` which means scrolling won't work anymore.

Also, if you're doing it this way you need to change out all instances of `BasicScrollContainer`. I don't see `CommandPanel` in this PR, so you probably have to address that and any more instances.

As it turns out osu! uses this scroll container override hack to get by the scroll problem: https://github.com/ppy/osu/blob/7654df94f6f37b8382be7dfcb4f674e03bd35427/osu.Game/Graphics/Containers/OsuScrollContainer.cs#L101

Reply:
I didn't think about that 😹. Ok, I added base.Onscroll() to it so it should be working fine now. Also switched out command panel.